include/boost/ut.hpp

```diff
@@ -1466,10 +1477,23 @@ template <class... Ts, class TEvent>
 struct test {
   utility::string_view type{};
   utility::string_view name{};
+  reflection::source_location location;
+
+  constexpr test(std::initializer_list<utility::string_view> l, const reflection::source_location& currentLocation =
```

Comment:
I'm afraid that this solution will always produce ut.hpp as the file_name :thinking: I believe to get it working we have to do it in the callee side, in ""_test in this case.

Reply:
Arg, I thought it work due to a cache of another include that filled the filename (it was for another test). I though I removed, but it's seems I didn't. When it worked I just pressumed UDL was simply syntax sugar and that the compiler was really calling the contructor directily. Not the case then. Forget about this attempt then :(